py/autoevals/llm.py

```diff
@@ -78,18 +78,21 @@ def __init__(
         render_args=None,
         max_tokens=None,
         temperature=None,
+        engine=None
     ):
         found = False
-        for m in SUPPORTED_MODELS:
-            # Prefixes are ok, because they are just time snapshots
-            if model.startswith(m):
-                found = True
-                break
-        if not found:
-            raise ValueError(f"Unsupported model: {model}. Currently only supports OpenAI chat models.")
+        if engine is None:
+            for m in SUPPORTED_MODELS:
+                # Prefixes are ok, because they are just time snapshots
+                if model.startswith(m):
+                    found = True
+                    break
+            if not found:
+                raise ValueError(f"Unsupported model: {model}. Currently only supports OpenAI chat models.")
```

Comment:
We may want to just remove this check altogether — we definitely support more than the listed models, and "non-chat" flavored models are all deprecated at this point.

Reply:
Removed